syntax = "proto3";

package payment;

message PaymentStatus {
  string order_id = 1;
  string user_id = 2;
  string payment_status = 3;
}
